app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(author_id: @user.id, post_id: @post.id)
+    return unless @like.save
+
+    redirect_to user_post_path(@user.id, @post)
+  end
+end
```

Comment:
- [ ] Please now that you have the Like button on the post index page, After I click the like button to create a new like, I get the error below so make sure that you fix this :pray: 
  
  ![Screenshot from 2023-07-28 16-05-43](https://github.com/BANCUNGUYE66/Blog-app/assets/92303891/c3ad7831-c859-42cf-ad1f-b4630bae3679)

Reply:
Thank you for your suggestion. but on my side everything is working, I can like and will not display any error.

Reply:
Thank you! but everything here is working well. check the video below

https://github.com/BANCUNGUYE66/Blog-app/assets/116639968/e989cfde-cfb6-4251-b85b-c66f5cef5f82